### ISSUE TITLE
[apostrophe-pages] Add unpublished floating menu to unpublished pages.

### ISSUE THREAD

**Issue by jth-** [punkave/apostrophe#534]
See #524 for design.

**Comment by mcoppola** [comment]
progress in 1cbedc4

**Comment by boutell** [comment]
As it is this would discard any page metadata that isn't part of the very limited pruned description of the page that we routinely push to the browser to avoid drowning it in JSON.

If you look at the page settings editor you'll note that it calls a route that, among other things, fetches a much more detailed representation of the page.

It would be a valid solution to call that same route to obtain the complete page object before updating it. And the more I look at it the more reasonable that sounds. Otherwise you would need a new route on the server side to update just the one property while respecting permissions, which seems like overkill.

```
return self.api(
        'fetch-to-update',
        {
          _id: self.page._id,
          type: self.page.type
        }, function(result) {
           // result.data is the page
        }
)
```

**Comment by mcoppola** [comment]
merged in 2ef8f6a
